src/Build.UnitTests/BackEnd/NodeEndpointInProc_Tests.cs

```diff
@@ -180,6 +182,18 @@ public void ConstructionWithValidHost()
                     NodeEndpointInProc.EndpointMode.Asynchronous, _host);
         }
 
+        [Fact]
+        public void HandshakesDiffer()
```

Comment:
Why this file? I would expect something more like `src\Build.UnitTests\Utilities_Tests.cs`